centos-bootc-config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "Labels": {
+        "org.opencontainers.image.redhat.rhel.compose-id": "CentOS-Stream-9-20240131.d.0",
+        "org.opencontainers.image.id": "CentOS-Stream",
```

Comment:
Also, let's aim to exactly match what is in the standard os-release metadata, instead of changing it:

```
$ podman run --rm -ti quay.io/centos/centos:stream9 grep -Ee '^ID' /etc/os-release
ID="centos"
ID_LIKE="rhel fedora"
$
```